### DISCUSSION TITLE
Hurtowa zmiana cen

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#6035] [Suggestions - in progress]
Możliwość zmiany cen wysyłając słownik z parami

```json
{
"id_aukcji": nowa_cena
}
```

**Comment by hahyes** [comment]
Jestem pewien, że taki potworek byłby niezgodny z założeniami REST API i RESTful. Nie mówiąc o tym, że przetwarzanie takiego potworka powodowałoby tylko ból głowy, bo trzeba to robić dynamicznie. Wiem, że są API, które tak robią - ale tak być nie powinno.

Moja propozycja, bardziej sensowna:
```json
[
  {
     "offerId": "id_aukcji",
     "price": nowa_cena
  },
  ...
]
```

**Comment by jakubmaguza** [comment]
Oczywiście masz rację :)

**Comment by morawcik** [comment]
Widzę, że raczej nie ma co liczyć na to :(

**Comment by jakubmaguza** [comment]
@MartaNowaczyk Są jakieś informacje na ten temat?

**Comment by MartaNowaczyk** [comment]
@jakubmaguza nie ma w tej chwili tego w planach i wątpię, żeby taka możliwość powstała. Na ten moment status zostawiam niezmieniony, potwierdzę to jeszcze z zespołem odpowiedzialnym za zmiany cen w ofercie.

**Comment by MartaNowaczyk** [comment]
@jakubmaguza @hahyes @morawcik wdrożyliśmy nowe zasoby do grupowej edycji ceny, które powinny zaopiekować ten temat #3179
